packages/shared/featureFlags.config.js

```diff
@@ -0,0 +1,43 @@
+const featureFlags = {
+    onboarding: {
+        enabled: true,
+    },
+    settings: {
+        enabled: true,
+    },
+    wallet: {
+        enabled: true,
+        accountSummaryAndAssets: {
+            enabled: true,
+        },
+        activityHistory: {
+            enabled: true,
+            sync: {
+                enabled: false,
+            },
+            search: {
+                enabled: true,
+            },
+            filter: {
```

Comment:
This is unused in the code. Not sure if you meant to include it in `AccountActivity.svelte`, though that part of the code is commented out.

Reply:
Ahh I was just creating an example. We will eventually add a filter to the component. I can wrap the current commented out code in this feature flag.